app/models/concerns/permitted_queries.rb

```diff
@@ -21,16 +21,23 @@ def initialize(groups, known_roles, is_admin)
   # @return [Array[String]] list of DRUIDs from APOs that this User can view
   def permitted_apos
     query = groups.map { |g| g.gsub(':', '\:') }.join(' OR ')
-    q = 'apo_role_group_manager_ssim:(' + query + ') OR apo_role_person_manager_ssim:(' + query + ')'
-    known_roles.each do |role|
-      q += ' OR apo_role_' + role + '_ssim:(' + query + ')'
+
+    q = ''
+    if is_admin
+      q += '*:*'
+    else
+      q += 'apo_role_group_manager_ssim:(' + query + ') OR apo_role_person_manager_ssim:(' + query + ')'
+      known_roles.each do |role|
+        q += ' OR apo_role_' + role + '_ssim:(' + query + ')'
+      end
     end
-    q = 'objectType_ssim:adminPolicy' if is_admin
+
     resp = repository.search(
       q: q,
+      defType: 'lucene',
       rows: 1000,
       fl: 'id',
-      fq: '!project_tag_ssim:"Hydrus"'
+      fq: ['objectType_ssim:adminPolicy', '!project_tag_ssim:"Hydrus"']
```

Comment:
Need to be? Probably not. Harmful? Not really. Anyway, that's how it was before.